cloudinit/config/schemas/schema-cloud-config-v1.json

```diff
@@ -292,9 +292,7 @@
           "type": "string"
         },
         "lock-passwd": {
-          "default": true,
           "type": "boolean",
-          "description": "Default: ``true``",
```

Comment:
+1 probably best not to describe a deprecated key.

Reply:
Yes I felt in the rendered docs it just gave wasted real-estate for duplicated description text to the large wall of text for documented keys. I've done the same dropping any documented "default" values for the deprecated keys as long as they have a suggested replacement which is documented.